.github/workflows/docker-image-perception-develop.yml

```diff
@@ -88,32 +88,74 @@ jobs:
         run: |
           docker push --all-tags ${GITHUB_REMOTE_URL}/${IMAGE_NAME}
 
-  notify-success:
-    name: "Notify Slack - Success"
+  build-ros-perception-cuda-x86-64:
+    name: "Build ROS perception CUDA X86/64"
     runs-on: ubuntu-latest
-    needs: [build-ros-perception-x86-64, build-ros-perception-arm64]
-    if: success()
+    env:
+      BASE_IMAGE: "docker.pkg.github.com/sonia-auv/sonia_common/sonia_common_cuda:x86-perception-latest"
+      ARCH: x86
+      TARGET_TYPE: perception
+      TARGET_VERSION: develop
+      IMAGE_NAME: proc_image_processing_cuda
+      GITHUB_REMOTE_URL: docker.pkg.github.com/${{ github.repository }}
     steps:
-      - name: Notify Slack Success
-        env:
-          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
-        uses: voxmedia/github-action-slack-notify-build@v1
-        with:
-          channel: github-ci-notifications
-          status: SUCCESS
-          color: good
-
-  notify-fail:
-    name: "Notify Slack - Failure"
+      - uses: actions/checkout@v2
+      - name: Login to Github Package Registry
+        run: |
+          echo "${{ secrets.GITHUB_TOKEN }}" | docker login docker.pkg.github.com -u ${{ github.actor }} --password-stdin
```

Comment:
remove spaces inside ${{ secrets.GITHUB_TOKEN }} and ${{ github.actor }}

Reply:
I'm following GitHub's syntax: https://docs.github.com/en/actions/reference/environment-variables

Reply:
It's also a copy-paste from what SONIA did